app/mailers/user_mailer.rb

```diff
@@ -36,4 +36,48 @@ def claim_created_support_notification(claim, user)
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
                          link_to_claim:))
   end
+
+  def partnership_created_notification(user, source_organisation, partner_organisation)
+    if partner_organisation.is_a?(Provider)
+      partner_class = "provider"
+      link = placements_provider_partner_schools_url(partner_organisation)
+    else
+      partner_class = "school"
+      link = placements_school_partner_providers_url(partner_organisation)
+    end
```

Comment:
Instead of using an `else` case, it might be easier to notice bugs if we use and `elsif` here.

With `else` cases, we could send in any other kind of object and it would try to be a `"school"`.